specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/examples/Apps_CreateOrUpdate.json

```diff
@@ -0,0 +1,88 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "resRg",
+    "resourceName": "myIoTCentralApp",
+    "api-version": "2021-11-01-preview",
+    "App": {
+      "location": "westus",
+      "sku": {
+        "name": "ST2"
+      },
+      "properties": {
+        "displayName": "My IoT Central App",
+        "subdomain": "my-iot-central-app",
+        "template": "iotc-pnp-preview@1.0.0"
+      },
+      "identity": {
+        "type": "SystemAssigned"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "header": {
+        "code": "200",
+        "message": "OK"
+      },
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.IoTCentral/IoTApps/myIoTCentralApp",
+        "name": "myIoTCentralApp",
+        "type": "Microsoft.IoTCentral/IoTApps",
+        "location": "westus",
+        "tags": {
+          "key": "value"
+        },
+        "properties": {
+          "applicationId": "6ebd8fd3-6e34-419e-908f-9be61ec6f6d6",
+          "displayName": "My IoT Central App 2",
+          "subdomain": "my-iot-central-app-2",
+          "template": "iotc-pnp-preview@1.0.0",
+          "state": "created"
+        },
+        "sku": {
+          "name": "ST2"
+        },
+        "identity": {
+          "type": "SystemAssigned",
+          "tenantId": "f686d426-8d16-42db-81b7-ab578e110ccd",
+          "principalId": "8988ab15-3e7a-4586-8a1c-ed07a73a53e9"
+        }
+      }
+    },
+    "201": {
+      "headers": {
+        "code": "201",
+        "message": "Created",
+        "target": "ResourceHandler.ExecutePutRequest"
+      },
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.IoTCentral/IoTApps/myIoTCentralApp",
+        "name": "myIoTCentralApp",
+        "type": "Microsoft.IoTCentral/IoTApps",
+        "location": "westus",
+        "tags": {
+          "key": "value"
+        },
+        "properties": {
+          "applicationId": "6ebd8fd3-6e34-419e-908f-9be61ec6f6d6",
+          "displayName": "My IoT Central App",
+          "subdomain": "my-iot-central-app",
+          "template": "iotc-pnp-preview@1.0.0",
+          "state": "created"
+        },
+        "sku": {
+          "name": "ST2"
+        },
+        "identity": {
+          "type": "SystemAssigned",
+          "tenantId": "f686d426-8d16-42db-81b7-ab578e110ccd",
+          "principalId": "8988ab15-3e7a-4586-8a1c-ed07a73a53e9"
+        }
+      }
+    },
+    "202": {
```

Comment:
We currently don't return a 202 for any of our PUT requests, only DELETE requests will return a 202